sound/soc/sof/sof-pci-dev.c

```diff
@@ -31,6 +31,10 @@ static int sof_pci_debug;
 module_param_named(sof_pci_debug, sof_pci_debug, int, 0444);
 MODULE_PARM_DESC(sof_pci_debug, "SOF PCI debug options (0x0 all off)");
 
+static int sof_pci_ipc_type = SOF_IPC;
```

Comment:
I think this should be `sof_pci_ipc_type = SOF_IPC_COUNT;`

Reply:
nope, that would cause out-of-bounds access.